component_descriptions/storage.md

```diff
@@ -0,0 +1,31 @@
+# Functional Component: storage
+
+## Specific scope of the module
+Collection items have storage locations. Representations of storage locations are necessary for collection management, in particular for all processing requiring information on the storage location of a given object and processes changing the location of an object, temporarily or permanently.
+
+## What does the module do? (envisioned feature set)
```

Comment:
Use term `component` instead of `module`

Reply:
OK.